scripts/download.js

```diff
@@ -552,9 +552,18 @@ function generateCode(){
 				var fileName = 'prism.' + type;
 
 				var codeElement = $('#download-' + type + ' code');
+				var pre = codeElement.parentElement;
 
-				codeElement.textContent = text;
-				Prism.highlightElement(codeElement, true);
+				var newCode = document.createElement('CODE');
+				newCode.className = codeElement.className;
+				newCode.textContent = text;
+
+				Prism.highlightElement(newCode, true, function () {
+					while (pre.firstChild) {
+						pre.removeChild(pre.firstChild);
+					}
+					pre.appendChild(newCode);
```

Comment:
Instead of clearing everything & appending, can we use `replaceChild` to swap the old with the new?

Reply:
Good idea!